data/layout/telescope_positions-North-TestLayout.ecsv

```diff
@@ -23,34 +23,26 @@
 #     value: -17.8920302
 #   center_northing: !astropy.units.Quantity
 #     unit: *id001
-#     value: 3185066.0
+#     value: 3185066.278
 #   corsika_obs_level: !astropy.units.Quantity
 #     unit: *id001
-#     value: 2158.0
+#     value: 2158
```

Comment:
21588 is good - do you remember where the MST telescopes heights have their origin? They are different to what are are using in prod6:
https://gitlab.cta-observatory.org/cta-science/simulations/simulation-and-performance-parallel-mode/layout-optimisation/layouts/-/blob/main/coordinates/telescope-list/North-Prod6-hyperArray-corsika.ecsv

Reply:
I hope you meant 2158 is good (not 21588). These were likely the values used for prod5. I updated the positions to match the prod6 layout